src/main/java/net/onelitefeather/bettergopaint/listeners/ConnectListener.java

```diff
@@ -26,15 +26,15 @@
 
 public class ConnectListener implements Listener {
 
-    public BetterGoPaint plugin;
+    private final BetterGoPaint plugin;
 
-    public ConnectListener(BetterGoPaint main) {
-        plugin = main;
+    public ConnectListener(BetterGoPaint plugin) {
+        this.plugin = plugin;
     }
 
     @EventHandler(priority = EventPriority.LOWEST)
-    public void onQuit(PlayerQuitEvent e) {
-        BetterGoPaint.getBrushManager().removePlayerBrush(e.getPlayer());
+    public void onQuit(PlayerQuitEvent event) {
+        plugin.getBrushManager().removeBrush(event.getPlayer());
```

Comment:
I do not recommend passing the entire main class when you only need to retrieve another class from it. For the sake of clarity, it is better to pass only the `BrushManager` to the constructor instead of the main class